include/net/xia_ether.h

```diff
@@ -0,0 +1,347 @@
+#ifndef _NET_XIA_ETHER_H
+#define _NET_XIA_ETHER_H
+/* prevents double declarations */
+
+#include <linux/netdevice.h>
+#include <linux/etherdevice.h>
+#include <linux/err.h>
+#include <net/xia_list_fib.h>
+#include <linux/netlink.h>
+#include <linux/spinlock.h>
+
+/* Ethernet Principal */
+#define XIDTYPE_ETHER (__cpu_to_be32(0x22))
+
+struct rtnl_xia_ether_addrs {
+	__u16		attr_len;
```

Comment:
For data structures, all the members should be aligned, except for some extremely long names. Also, please use ``space`` instead of ``tab`` for alignment.